### PR TITLE
Update DG by replacing old diagrams and adding Planned Enhancement Section

### PR DESCRIPTION
**Pull request by peaol**
== ready to merge ==

Changes:
- Replaced old diagrams that contained 'AddressBook' to 'Roster' and 'Person' to 'Student' and changed diagram fields to that of Student Command ones
- Added a Planned Enhancement Section

### CODE REVIEW

File: docs/images/ArchitectureSequenceDiagram.png
Comment (tyx021) [inline docs/images/ArchitectureSequenceDiagram.png:1]: should be `delete student 1`?

File: docs/images/DeleteSequenceDiagram.png
Comment (tyx021) [inline docs/images/DeleteSequenceDiagram.png:1]: Similar issue

Comment (tyx021) [reply docs/images/DeleteSequenceDiagram.png:1]: and `DeleteCommand` -> `DeleteStudentCommand`?

File: docs/images/ModelClassDiagram.png
Comment (tyx021) [inline docs/images/ModelClassDiagram.png:1]: Should we include `Remark`?

**Review by tyx021 — COMMENTED** [review]
👍👍👍

**Comment by tyx021** [comment]
Fixes #183

File: docs/images/DeleteSequenceDiagram.png
Comment (IsabelChong) [reply docs/images/DeleteSequenceDiagram.png:1]: Should it be `deleteStudent(s)` instead of `deletePerson(1)`?

File: docs/images/ModelClassDiagram.png
Comment (IsabelChong) [reply docs/images/ModelClassDiagram.png:1]: Following our naming conventions, we should name it `StudentId` instead of `SID`.

**Review by IsabelChong — COMMENTED** [review]
Great job! 👍 Just some minor changes

File: docs/images/DeleteSequenceDiagram.png
Comment (tyx021) [reply docs/images/DeleteSequenceDiagram.png:1]: image
Yes, I think it should be deleteStudent(s)

**Comment by IsabelChong** [comment]
LGTM! good job

**Pull request merged**